File: ws-eclipse/projetos_udemy/curso_udemy/src/soma_impares.java
import java.util.Locale;
import java.util.Scanner;

public class soma_impares {
public static void soma_impares(String[] args) {
	
 Locale.setDefault(Locale.US);
 Scanner sc = new Scanner(System.in);
 
 int n1, n2, soma, impar, impar2;
 
 System.out.println("Digite o primeiro numero: ");
 n1 = sc.nextInt();
 
 System.out.println("Digite o segundo numero : ");
 n2 = sc.nextInt();
 
 for(n1 % 2 != 0) {
	 impar = n1;
 }
 
 for(n2 % 2 != 0) {
	 impar2 = n2;
 }
 
 System.out.println("SOMA DOS IMPARES : " + soma);

	}

}

File: ws-codeblocks/projetos_udemy_C++/soma_impares.cpp
#include <bits/stdc++.h>

using namespace std;

int main()
{

int numeroum, numerodois, impar, impardois, soma;

cout("Digite o primeiro numero: ");
cin(numeroum);

cout("Digite o segundo numero : ");
cin(numerodois);

if (numeroum % 2 != 0){
    impar = numeroum;
}

if (numerodois % 2 != 0){
    impardois = numerodois;
}

soma = impar + impardois;

cout("SOMA DOS IMPARES \n" <<soma);





return 0;
}
